### ISSUE TITLE
Uncaught ReferenceError: BroadcastChannel is not defined

### ISSUE THREAD

**Issue by jsejcksn** [denoland/deno#13214]
`BroadcastChannel`:

- is listed in the [stable](https://doc.deno.land/deno/stable@v1.17.1/~/BroadcastChannel) docs
- is not listed in the [unstable](https://doc.deno.land/deno/unstable@v1.17.1) docs
- throws a `ReferenceError` without using `--unstable`:

    ```
    $ deno repl
    Deno 1.17.1
    exit using ctrl+d or close()
    > new BroadcastChannel()
    Uncaught ReferenceError: BroadcastChannel is not defined
        at <anonymous>:2:1
    > close()

    $ deno repl --unstable
    Deno 1.17.1
    exit using ctrl+d or close()
    > new BroadcastChannel()
    Uncaught TypeError: Failed to construct 'BroadcastChannel': 1 argument required, but only 0 present.
        at Object.requiredArguments (deno:ext/webidl/00_webidl.js:627:13)
        at new BroadcastChannel (deno:ext/broadcast_channel/01_broadcast_channel.js:81:14)
        at <anonymous>:2:1
    > close()
    ```

Is this a bug or just a docs problem? I searched through several pages of release notes and found only this mention in [`v1.11.0`](https://github.com/denoland/deno/releases/tag/v1.11.0):

> - feat(extensions): add BroadcastChannel

**Comment by bnoordhuis** [comment]
It's a documentation issue, BroadcastChannel is still an unstable feature.

I suspect it's caused by the inclusion of `ext/broadcast_channel/lib.deno_broadcast_channel.d.ts` in `cli/dts/lib.deno.shared_globals.d.ts` (BroadcastChannel is a browser global.)

**Comment by emschwartz** [comment]
I'm also seeing `ReferenceError: BroadcastChannel is not defined` using Deno 1.26.0

**Comment by emschwartz** [comment]
I'm curious about the actual (in)stability of this feature, given that it's supported on Deno Deploy 🤔 https://deno.com/deploy/docs/runtime-broadcast-channel

**Comment by jsejcksn** [comment]
Closed by [#19108](https://github.com/denoland/deno/pull/19108)
